mappings/net/minecraft/nbt/NbtCompound.mapping

```diff
@@ -26,6 +32,7 @@ CLASS net/minecraft/class_2487 net/minecraft/nbt/CompoundTag
 		ARG 1 key
 		ARG 2 value
 	METHOD method_10545 contains (Ljava/lang/String;)Z
+		COMMENT Returns whether the compound object contains the specified key.
```

Comment:
This should be consistent with other documentation:

```suggestion
		COMMENT Returns whether the NBT compound object contains the specified key.
```

Is there any particular reason why 'NBT compound object' is used and not 'NBT compound'?